templates/submission/wizard.tpl

```diff
@@ -19,7 +19,7 @@
             </template>
             <template v-if="localize(publication.title)">
                 <span class="app__breadcrumbsSeparator" aria-hidden="true">/</span>
-                {{ localize(publication.title) }}
+                <span v-html="localize(publication.title)">
```

Comment:
I wonder what your opinion is, @NateWr -- but even though it's possibly redundant, I would prefer we enforced the limited HTML support for title fields again in JS-land rather than blankly trusting the HTML that's provided. Maybe run it through something like https://www.npmjs.com/package/striptags?

Reply:
Personally, I'd prefer to leave this sort of thing to the responsibility of the server. Another way to put it is that we should be able to trust data we get from the server. Putting sanitization into the browser layer will introduce a lot of uncertainty or cruft in that layer, which I think our team is better equipped to manage on the server.

If it makes you feel any better, `v-html` with a malicious string like `<script>` will not execute. It sets the `innerHTML` property, which won't download and execute a `<script>` tag. So while it would potentially support HTML we don't like, it should protect against the most dangerous strings.

Reply:
:+1: :+1: That makes sense to me (and I'm glad to hear about `<script>` being specifically declawed).